reference/configuration/swiftmailer.rst

```diff
@@ -156,6 +157,15 @@ emails sent during development go to a single account.
 This uses ``Swift_Plugins_RedirectingPlugin``. Original recipients are available
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
+~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Used in combination with ``delivery_address``. If set, emails matching any of these
+patterns will be delivered like normal, instead of being sent to `delivery_address`.
```

Comment:
sorry, one last thing: you have to use double backticks for inline codeblocks in reStructured text.